[2024-06-20T11:43:19.630+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-20T11:43:19.675+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T11:43:19.684+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T11:43:19.685+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-06-20T11:43:19.698+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_run> on 2024-06-18 00:00:00+00:00
[2024-06-20T11:43:19.715+0000] {logging_mixin.py:188} WARNING - /home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=11134) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-20T11:43:19.716+0000] {standard_task_runner.py:63} INFO - Started process 11251 to run task
[2024-06-20T11:43:19.715+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nba_data_pipeline_dag', 'dbt_run', 'scheduled__2024-06-18T00:00:00+00:00', '--job-id', '856', '--raw', '--subdir', 'DAGS_FOLDER/nba_data_pipeline_dag.py', '--cfg-path', '/tmp/tmptmscafra']
[2024-06-20T11:43:19.720+0000] {standard_task_runner.py:91} INFO - Job 856: Subtask dbt_run
[2024-06-20T11:43:19.827+0000] {task_command.py:426} INFO - Running <TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [running]> on host ip-172-31-45-144.ec2.internal
[2024-06-20T11:43:19.989+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='nba_data_pipeline_dag' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-18T00:00:00+00:00'
[2024-06-20T11:43:19.993+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-20T11:43:20.007+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-20T11:43:20.008+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /home/ubuntu/nba_project && dbt run']
[2024-06-20T11:43:20.024+0000] {subprocess.py:86} INFO - Output:
[2024-06-20T11:43:22.524+0000] {subprocess.py:93} INFO - [0m11:43:22  Running with dbt=1.8.2
[2024-06-20T11:43:23.725+0000] {subprocess.py:93} INFO - [0m11:43:23  Registered adapter: snowflake=1.8.3
[2024-06-20T11:43:23.755+0000] {subprocess.py:93} INFO - [0m11:43:23  Unable to do partial parsing because saved manifest not found. Starting full parse.
[2024-06-20T11:43:25.369+0000] {subprocess.py:93} INFO - [0m11:43:25  [[33mWARNING[0m]: Did not find matching node for patch with name 'staging_player_stats' in the 'models' section of file 'models/schema.yml'
[2024-06-20T11:43:25.379+0000] {subprocess.py:93} INFO - [0m11:43:25  [[33mWARNING[0m]: Did not find matching node for patch with name 'staging_nba_scores' in the 'models' section of file 'models/schema.yml'
[2024-06-20T11:43:25.384+0000] {subprocess.py:93} INFO - [0m11:43:25  [[33mWARNING[0m]: Did not find matching node for patch with name 'staging_nba_news' in the 'models' section of file 'models/schema.yml'
[2024-06-20T11:43:25.387+0000] {subprocess.py:93} INFO - [0m11:43:25  [[33mWARNING[0m]: Did not find matching node for patch with name 'staging_top_100_players_stats' in the 'models' section of file 'models/schema.yml'
[2024-06-20T11:43:25.543+0000] {subprocess.py:93} INFO - [0m11:43:25  Encountered an error:
[2024-06-20T11:43:25.544+0000] {subprocess.py:93} INFO - Compilation Error
[2024-06-20T11:43:25.545+0000] {subprocess.py:93} INFO -   Model 'model.nba_project.player_stats' (models/staging/player_stats.sql) depends on a source named 'nba_stats.player_stats' which was not found
[2024-06-20T11:43:26.249+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2024-06-20T11:43:26.250+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-20T11:43:26.278+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2024-06-20T11:43:26.284+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=nba_data_pipeline_dag, task_id=dbt_run, run_id=scheduled__2024-06-18T00:00:00+00:00, execution_date=20240618T000000, start_date=20240620T114319, end_date=20240620T114326
[2024-06-20T11:43:26.298+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 856 for task dbt_run (Bash command failed. The command returned a non-zero exit code 2.; 11251)
[2024-06-20T11:43:26.321+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-20T11:43:26.354+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-20T11:43:26.356+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-20T11:55:18.033+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-20T11:55:18.083+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T11:55:18.093+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T11:55:18.093+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-06-20T11:55:18.112+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_run> on 2024-06-18 00:00:00+00:00
[2024-06-20T11:55:18.133+0000] {logging_mixin.py:188} WARNING - /home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=17645) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-20T11:55:18.137+0000] {standard_task_runner.py:63} INFO - Started process 17697 to run task
[2024-06-20T11:55:18.136+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nba_data_pipeline_dag', 'dbt_run', 'scheduled__2024-06-18T00:00:00+00:00', '--job-id', '893', '--raw', '--subdir', 'DAGS_FOLDER/nba_data_pipeline_dag.py', '--cfg-path', '/tmp/tmpcmofgul1']
[2024-06-20T11:55:18.142+0000] {standard_task_runner.py:91} INFO - Job 893: Subtask dbt_run
[2024-06-20T11:55:18.296+0000] {task_command.py:426} INFO - Running <TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [running]> on host ip-172-31-45-144.ec2.internal
[2024-06-20T11:55:18.460+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='nba_data_pipeline_dag' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-18T00:00:00+00:00'
[2024-06-20T11:55:18.462+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-20T11:55:18.477+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-20T11:55:18.478+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /home/ubuntu/nba_project && dbt run']
[2024-06-20T11:55:18.492+0000] {subprocess.py:86} INFO - Output:
[2024-06-20T11:55:20.985+0000] {subprocess.py:93} INFO - [0m11:55:20  Running with dbt=1.8.2
[2024-06-20T11:55:22.211+0000] {subprocess.py:93} INFO - [0m11:55:22  Registered adapter: snowflake=1.8.3
[2024-06-20T11:55:22.236+0000] {subprocess.py:93} INFO - [0m11:55:22  Unable to do partial parsing because saved manifest not found. Starting full parse.
[2024-06-20T11:55:24.134+0000] {subprocess.py:93} INFO - [0m11:55:24  Encountered an error:
[2024-06-20T11:55:24.135+0000] {subprocess.py:93} INFO - Compilation Error
[2024-06-20T11:55:24.135+0000] {subprocess.py:93} INFO -   Model 'model.nba_project.staging_player_stats' (models/staging/staging_player_stats.sql) depends on a source named 'nba_stats.player_stats' which was not found
[2024-06-20T11:55:24.791+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2024-06-20T11:55:24.792+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-20T11:55:24.819+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2024-06-20T11:55:24.824+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=nba_data_pipeline_dag, task_id=dbt_run, run_id=scheduled__2024-06-18T00:00:00+00:00, execution_date=20240618T000000, start_date=20240620T115518, end_date=20240620T115524
[2024-06-20T11:55:24.839+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 893 for task dbt_run (Bash command failed. The command returned a non-zero exit code 2.; 17697)
[2024-06-20T11:55:24.868+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-20T11:55:24.912+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-20T11:55:24.914+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-20T12:15:19.594+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-20T12:15:19.656+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T12:15:19.672+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T12:15:19.673+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-06-20T12:15:19.703+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_run> on 2024-06-18 00:00:00+00:00
[2024-06-20T12:15:19.718+0000] {logging_mixin.py:188} WARNING - /home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=26208) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-20T12:15:19.720+0000] {standard_task_runner.py:63} INFO - Started process 26275 to run task
[2024-06-20T12:15:19.718+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nba_data_pipeline_dag', 'dbt_run', 'scheduled__2024-06-18T00:00:00+00:00', '--job-id', '941', '--raw', '--subdir', 'DAGS_FOLDER/nba_data_pipeline_dag.py', '--cfg-path', '/tmp/tmpwce4sd4k']
[2024-06-20T12:15:19.724+0000] {standard_task_runner.py:91} INFO - Job 941: Subtask dbt_run
[2024-06-20T12:15:19.838+0000] {task_command.py:426} INFO - Running <TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [running]> on host ip-172-31-45-144.ec2.internal
[2024-06-20T12:15:20.021+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='nba_data_pipeline_dag' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-18T00:00:00+00:00'
[2024-06-20T12:15:20.024+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-20T12:15:20.039+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-20T12:15:20.040+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /home/ubuntu/nba_project && dbt run']
[2024-06-20T12:15:20.056+0000] {subprocess.py:86} INFO - Output:
[2024-06-20T12:15:22.573+0000] {subprocess.py:93} INFO - [0m12:15:22  Running with dbt=1.8.2
[2024-06-20T12:15:23.731+0000] {subprocess.py:93} INFO - [0m12:15:23  Registered adapter: snowflake=1.8.3
[2024-06-20T12:15:24.124+0000] {subprocess.py:93} INFO - [0m12:15:24  Found 8 models, 4 sources, 444 macros
[2024-06-20T12:15:24.130+0000] {subprocess.py:93} INFO - [0m12:15:24
[2024-06-20T12:15:24.409+0000] {subprocess.py:93} INFO - [0m12:15:24
[2024-06-20T12:15:24.410+0000] {subprocess.py:93} INFO - [0m12:15:24  Finished running  in 0 hours 0 minutes and 0.28 seconds (0.28s).
[2024-06-20T12:15:24.410+0000] {subprocess.py:93} INFO - [0m12:15:24  Encountered an error:
[2024-06-20T12:15:24.410+0000] {subprocess.py:93} INFO - Runtime Error
[2024-06-20T12:15:24.411+0000] {subprocess.py:93} INFO -   Database error while listing schemas in database "NBA_STATS_ANALYTICS"
[2024-06-20T12:15:24.411+0000] {subprocess.py:93} INFO -   Database Error
[2024-06-20T12:15:24.411+0000] {subprocess.py:93} INFO -     250001 (08001): Failed to connect to DB: lv27483.us-east-2.aws.snowflakecomputing.com:443. Role 'DEV' specified in the connect string does not exist or not authorized. Contact your local system administrator, or attempt to login with another role, e.g. PUBLIC.
[2024-06-20T12:15:26.106+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2024-06-20T12:15:26.107+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-20T12:15:26.134+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2024-06-20T12:15:26.139+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=nba_data_pipeline_dag, task_id=dbt_run, run_id=scheduled__2024-06-18T00:00:00+00:00, execution_date=20240618T000000, start_date=20240620T121519, end_date=20240620T121526
[2024-06-20T12:15:26.154+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 941 for task dbt_run (Bash command failed. The command returned a non-zero exit code 2.; 26275)
[2024-06-20T12:15:26.167+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-20T12:15:26.199+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-20T12:15:26.204+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-20T12:23:54.449+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-20T12:23:54.495+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T12:23:54.504+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T12:23:54.505+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-06-20T12:23:54.518+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_run> on 2024-06-18 00:00:00+00:00
[2024-06-20T12:23:54.528+0000] {logging_mixin.py:188} WARNING - /home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=32380) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-20T12:23:54.530+0000] {standard_task_runner.py:63} INFO - Started process 32388 to run task
[2024-06-20T12:23:54.529+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nba_data_pipeline_dag', 'dbt_run', 'scheduled__2024-06-18T00:00:00+00:00', '--job-id', '976', '--raw', '--subdir', 'DAGS_FOLDER/nba_data_pipeline_dag.py', '--cfg-path', '/tmp/tmpil5b6l55']
[2024-06-20T12:23:54.532+0000] {standard_task_runner.py:91} INFO - Job 976: Subtask dbt_run
[2024-06-20T12:23:54.637+0000] {task_command.py:426} INFO - Running <TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [running]> on host ip-172-31-45-144.ec2.internal
[2024-06-20T12:23:54.798+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='nba_data_pipeline_dag' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-18T00:00:00+00:00'
[2024-06-20T12:23:54.799+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-20T12:23:54.813+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-20T12:23:54.814+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /home/ubuntu/nba_project && dbt run']
[2024-06-20T12:23:54.827+0000] {subprocess.py:86} INFO - Output:
[2024-06-20T12:23:56.938+0000] {subprocess.py:93} INFO - [0m12:23:56  Running with dbt=1.8.2
[2024-06-20T12:23:57.981+0000] {subprocess.py:93} INFO - [0m12:23:57  Registered adapter: snowflake=1.8.3
[2024-06-20T12:23:58.286+0000] {subprocess.py:93} INFO - [0m12:23:58  Found 8 models, 4 sources, 444 macros
[2024-06-20T12:23:58.290+0000] {subprocess.py:93} INFO - [0m12:23:58
[2024-06-20T12:24:02.701+0000] {subprocess.py:93} INFO - [0m12:24:02  Concurrency: 1 threads (target='dev')
[2024-06-20T12:24:02.702+0000] {subprocess.py:93} INFO - [0m12:24:02
[2024-06-20T12:24:02.717+0000] {subprocess.py:93} INFO - [0m12:24:02  1 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_nba_news ........... [RUN]
[2024-06-20T12:24:03.291+0000] {subprocess.py:93} INFO - [0m12:24:03  1 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_nba_news ...... [[32mSUCCESS 1[0m in 0.57s]
[2024-06-20T12:24:03.295+0000] {subprocess.py:93} INFO - [0m12:24:03  2 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_nba_scores ......... [RUN]
[2024-06-20T12:24:03.794+0000] {subprocess.py:93} INFO - [0m12:24:03  2 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_nba_scores .... [[32mSUCCESS 1[0m in 0.50s]
[2024-06-20T12:24:03.796+0000] {subprocess.py:93} INFO - [0m12:24:03  3 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_player_stats ....... [RUN]
[2024-06-20T12:24:04.487+0000] {subprocess.py:93} INFO - [0m12:24:04  3 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_player_stats .. [[32mSUCCESS 1[0m in 0.69s]
[2024-06-20T12:24:04.489+0000] {subprocess.py:93} INFO - [0m12:24:04  4 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_top_100_players_stats  [RUN]
[2024-06-20T12:24:04.943+0000] {subprocess.py:93} INFO - [0m12:24:04  4 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_top_100_players_stats  [[32mSUCCESS 1[0m in 0.45s]
[2024-06-20T12:24:04.946+0000] {subprocess.py:93} INFO - [0m12:24:04  5 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_nba_news ...... [RUN]
[2024-06-20T12:24:08.430+0000] {subprocess.py:93} INFO - [0m12:24:08  5 of 8 ERROR creating sql table model NBA_STATS_dbt_nba_stats.transformed_nba_news  [[31mERROR[0m in 3.48s]
[2024-06-20T12:24:08.433+0000] {subprocess.py:93} INFO - [0m12:24:08  6 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_nba_scores .... [RUN]
[2024-06-20T12:24:09.757+0000] {subprocess.py:93} INFO - [0m12:24:09  6 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_nba_scores  [[32mSUCCESS 1[0m in 1.32s]
[2024-06-20T12:24:09.760+0000] {subprocess.py:93} INFO - [0m12:24:09  7 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_player_stats .. [RUN]
[2024-06-20T12:24:13.409+0000] {subprocess.py:93} INFO - [0m12:24:13  7 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_player_stats  [[32mSUCCESS 1[0m in 3.65s]
[2024-06-20T12:24:13.411+0000] {subprocess.py:93} INFO - [0m12:24:13  8 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_top_100_players_stats  [RUN]
[2024-06-20T12:24:14.787+0000] {subprocess.py:93} INFO - [0m12:24:14  8 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_top_100_players_stats  [[32mSUCCESS 1[0m in 1.37s]
[2024-06-20T12:24:14.791+0000] {subprocess.py:93} INFO - [0m12:24:14
[2024-06-20T12:24:14.792+0000] {subprocess.py:93} INFO - [0m12:24:14  Finished running 4 view models, 4 table models in 0 hours 0 minutes and 16.50 seconds (16.50s).
[2024-06-20T12:24:14.838+0000] {subprocess.py:93} INFO - [0m12:24:14
[2024-06-20T12:24:14.839+0000] {subprocess.py:93} INFO - [0m12:24:14  [31mCompleted with 1 error and 0 warnings:[0m
[2024-06-20T12:24:14.839+0000] {subprocess.py:93} INFO - [0m12:24:14
[2024-06-20T12:24:14.840+0000] {subprocess.py:93} INFO - [0m12:24:14    Database Error in model transformed_nba_news (models/transformed/transformed_nba_news.sql)
[2024-06-20T12:24:14.840+0000] {subprocess.py:93} INFO -   000904 (42000): SQL compilation error: error line 9 at position 8
[2024-06-20T12:24:14.840+0000] {subprocess.py:93} INFO -   invalid identifier 'SOURCE'
[2024-06-20T12:24:14.840+0000] {subprocess.py:93} INFO -   compiled Code at target/run/nba_project/models/transformed/transformed_nba_news.sql
[2024-06-20T12:24:14.840+0000] {subprocess.py:93} INFO - [0m12:24:14
[2024-06-20T12:24:14.841+0000] {subprocess.py:93} INFO - [0m12:24:14  Done. PASS=7 WARN=0 ERROR=1 SKIP=0 TOTAL=8
[2024-06-20T12:24:15.580+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-06-20T12:24:15.581+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-20T12:24:15.609+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-06-20T12:24:15.614+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=nba_data_pipeline_dag, task_id=dbt_run, run_id=scheduled__2024-06-18T00:00:00+00:00, execution_date=20240618T000000, start_date=20240620T122354, end_date=20240620T122415
[2024-06-20T12:24:15.628+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 976 for task dbt_run (Bash command failed. The command returned a non-zero exit code 1.; 32388)
[2024-06-20T12:24:15.671+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-20T12:24:15.703+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-20T12:24:15.705+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-20T12:42:54.357+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-20T12:42:54.403+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T12:42:54.413+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T12:42:54.413+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-06-20T12:42:54.428+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_run> on 2024-06-18 00:00:00+00:00
[2024-06-20T12:42:54.445+0000] {logging_mixin.py:188} WARNING - /home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=39958) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-20T12:42:54.449+0000] {standard_task_runner.py:63} INFO - Started process 40027 to run task
[2024-06-20T12:42:54.444+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nba_data_pipeline_dag', 'dbt_run', 'scheduled__2024-06-18T00:00:00+00:00', '--job-id', '1027', '--raw', '--subdir', 'DAGS_FOLDER/nba_data_pipeline_dag.py', '--cfg-path', '/tmp/tmpiz5_7n43']
[2024-06-20T12:42:54.450+0000] {standard_task_runner.py:91} INFO - Job 1027: Subtask dbt_run
[2024-06-20T12:42:54.559+0000] {task_command.py:426} INFO - Running <TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [running]> on host ip-172-31-45-144.ec2.internal
[2024-06-20T12:42:54.768+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='nba_data_pipeline_dag' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-18T00:00:00+00:00'
[2024-06-20T12:42:54.769+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-20T12:42:54.786+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-20T12:42:54.786+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /home/ubuntu/nba_project && dbt run']
[2024-06-20T12:42:54.801+0000] {subprocess.py:86} INFO - Output:
[2024-06-20T12:42:57.051+0000] {subprocess.py:93} INFO - [0m12:42:57  Running with dbt=1.8.2
[2024-06-20T12:42:58.161+0000] {subprocess.py:93} INFO - [0m12:42:58  Registered adapter: snowflake=1.8.3
[2024-06-20T12:42:58.534+0000] {subprocess.py:93} INFO - [0m12:42:58  Found 8 models, 4 sources, 444 macros
[2024-06-20T12:42:58.540+0000] {subprocess.py:93} INFO - [0m12:42:58
[2024-06-20T12:43:04.441+0000] {subprocess.py:93} INFO - [0m12:43:04  Concurrency: 1 threads (target='dev')
[2024-06-20T12:43:04.442+0000] {subprocess.py:93} INFO - [0m12:43:04
[2024-06-20T12:43:04.448+0000] {subprocess.py:93} INFO - [0m12:43:04  1 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_nba_news ........... [RUN]
[2024-06-20T12:43:05.065+0000] {subprocess.py:93} INFO - [0m12:43:05  1 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_nba_news ...... [[32mSUCCESS 1[0m in 0.61s]
[2024-06-20T12:43:05.067+0000] {subprocess.py:93} INFO - [0m12:43:05  2 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_nba_scores ......... [RUN]
[2024-06-20T12:43:05.503+0000] {subprocess.py:93} INFO - [0m12:43:05  2 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_nba_scores .... [[32mSUCCESS 1[0m in 0.43s]
[2024-06-20T12:43:05.506+0000] {subprocess.py:93} INFO - [0m12:43:05  3 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_player_stats ....... [RUN]
[2024-06-20T12:43:05.975+0000] {subprocess.py:93} INFO - [0m12:43:05  3 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_player_stats .. [[32mSUCCESS 1[0m in 0.47s]
[2024-06-20T12:43:05.978+0000] {subprocess.py:93} INFO - [0m12:43:05  4 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_top_100_players_stats  [RUN]
[2024-06-20T12:43:06.794+0000] {subprocess.py:93} INFO - [0m12:43:06  4 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_top_100_players_stats  [[32mSUCCESS 1[0m in 0.81s]
[2024-06-20T12:43:06.797+0000] {subprocess.py:93} INFO - [0m12:43:06  5 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_nba_news ...... [RUN]
[2024-06-20T12:43:07.537+0000] {subprocess.py:93} INFO - [0m12:43:07  5 of 8 ERROR creating sql table model NBA_STATS_dbt_nba_stats.transformed_nba_news  [[31mERROR[0m in 0.74s]
[2024-06-20T12:43:07.540+0000] {subprocess.py:93} INFO - [0m12:43:07  6 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_nba_scores .... [RUN]
[2024-06-20T12:43:08.442+0000] {subprocess.py:93} INFO - [0m12:43:08  6 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_nba_scores  [[32mSUCCESS 1[0m in 0.90s]
[2024-06-20T12:43:08.444+0000] {subprocess.py:93} INFO - [0m12:43:08  7 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_player_stats .. [RUN]
[2024-06-20T12:43:09.331+0000] {subprocess.py:93} INFO - [0m12:43:09  7 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_player_stats  [[32mSUCCESS 1[0m in 0.88s]
[2024-06-20T12:43:09.333+0000] {subprocess.py:93} INFO - [0m12:43:09  8 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_top_100_players_stats  [RUN]
[2024-06-20T12:43:10.731+0000] {subprocess.py:93} INFO - [0m12:43:10  8 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_top_100_players_stats  [[32mSUCCESS 1[0m in 1.40s]
[2024-06-20T12:43:10.735+0000] {subprocess.py:93} INFO - [0m12:43:10
[2024-06-20T12:43:10.736+0000] {subprocess.py:93} INFO - [0m12:43:10  Finished running 4 view models, 4 table models in 0 hours 0 minutes and 12.19 seconds (12.19s).
[2024-06-20T12:43:10.915+0000] {subprocess.py:93} INFO - [0m12:43:10
[2024-06-20T12:43:10.916+0000] {subprocess.py:93} INFO - [0m12:43:10  [31mCompleted with 1 error and 0 warnings:[0m
[2024-06-20T12:43:10.916+0000] {subprocess.py:93} INFO - [0m12:43:10
[2024-06-20T12:43:10.917+0000] {subprocess.py:93} INFO - [0m12:43:10    Database Error in model transformed_nba_news (models/transformed/transformed_nba_news.sql)
[2024-06-20T12:43:10.917+0000] {subprocess.py:93} INFO -   000904 (42000): SQL compilation error: error line 9 at position 8
[2024-06-20T12:43:10.917+0000] {subprocess.py:93} INFO -   invalid identifier '"source"'
[2024-06-20T12:43:10.918+0000] {subprocess.py:93} INFO -   compiled Code at target/run/nba_project/models/transformed/transformed_nba_news.sql
[2024-06-20T12:43:10.918+0000] {subprocess.py:93} INFO - [0m12:43:10
[2024-06-20T12:43:10.918+0000] {subprocess.py:93} INFO - [0m12:43:10  Done. PASS=7 WARN=0 ERROR=1 SKIP=0 TOTAL=8
[2024-06-20T12:43:11.598+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-06-20T12:43:11.599+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-20T12:43:11.626+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-06-20T12:43:11.631+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=nba_data_pipeline_dag, task_id=dbt_run, run_id=scheduled__2024-06-18T00:00:00+00:00, execution_date=20240618T000000, start_date=20240620T124254, end_date=20240620T124311
[2024-06-20T12:43:11.646+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 1027 for task dbt_run (Bash command failed. The command returned a non-zero exit code 1.; 40027)
[2024-06-20T12:43:11.660+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-20T12:43:11.692+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-20T12:43:11.694+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-20T12:52:24.586+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-20T12:52:24.635+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T12:52:24.646+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T12:52:24.646+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-06-20T12:52:24.664+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_run> on 2024-06-18 00:00:00+00:00
[2024-06-20T12:52:24.678+0000] {logging_mixin.py:188} WARNING - /home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=46232) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-20T12:52:24.681+0000] {standard_task_runner.py:63} INFO - Started process 46318 to run task
[2024-06-20T12:52:24.679+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nba_data_pipeline_dag', 'dbt_run', 'scheduled__2024-06-18T00:00:00+00:00', '--job-id', '1063', '--raw', '--subdir', 'DAGS_FOLDER/nba_data_pipeline_dag.py', '--cfg-path', '/tmp/tmp69xvurik']
[2024-06-20T12:52:24.684+0000] {standard_task_runner.py:91} INFO - Job 1063: Subtask dbt_run
[2024-06-20T12:52:24.784+0000] {task_command.py:426} INFO - Running <TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [running]> on host ip-172-31-45-144.ec2.internal
[2024-06-20T12:52:24.940+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='nba_data_pipeline_dag' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-18T00:00:00+00:00'
[2024-06-20T12:52:24.941+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-20T12:52:24.956+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-20T12:52:24.956+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /home/ubuntu/nba_project && dbt run']
[2024-06-20T12:52:24.970+0000] {subprocess.py:86} INFO - Output:
[2024-06-20T12:52:27.169+0000] {subprocess.py:93} INFO - [0m12:52:27  Running with dbt=1.8.2
[2024-06-20T12:52:28.247+0000] {subprocess.py:93} INFO - [0m12:52:28  Registered adapter: snowflake=1.8.3
[2024-06-20T12:52:28.574+0000] {subprocess.py:93} INFO - [0m12:52:28  Found 8 models, 4 sources, 444 macros
[2024-06-20T12:52:28.579+0000] {subprocess.py:93} INFO - [0m12:52:28
[2024-06-20T12:52:29.549+0000] {subprocess.py:93} INFO - [0m12:52:29  Concurrency: 1 threads (target='dev')
[2024-06-20T12:52:29.550+0000] {subprocess.py:93} INFO - [0m12:52:29
[2024-06-20T12:52:29.554+0000] {subprocess.py:93} INFO - [0m12:52:29  1 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_nba_news ........... [RUN]
[2024-06-20T12:52:30.200+0000] {subprocess.py:93} INFO - [0m12:52:30  1 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_nba_news ...... [[32mSUCCESS 1[0m in 0.64s]
[2024-06-20T12:52:30.202+0000] {subprocess.py:93} INFO - [0m12:52:30  2 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_nba_scores ......... [RUN]
[2024-06-20T12:52:30.652+0000] {subprocess.py:93} INFO - [0m12:52:30  2 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_nba_scores .... [[32mSUCCESS 1[0m in 0.45s]
[2024-06-20T12:52:30.654+0000] {subprocess.py:93} INFO - [0m12:52:30  3 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_player_stats ....... [RUN]
[2024-06-20T12:52:32.649+0000] {subprocess.py:93} INFO - [0m12:52:32  3 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_player_stats .. [[32mSUCCESS 1[0m in 1.99s]
[2024-06-20T12:52:32.651+0000] {subprocess.py:93} INFO - [0m12:52:32  4 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_top_100_players_stats  [RUN]
[2024-06-20T12:52:33.188+0000] {subprocess.py:93} INFO - [0m12:52:33  4 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_top_100_players_stats  [[32mSUCCESS 1[0m in 0.53s]
[2024-06-20T12:52:33.190+0000] {subprocess.py:93} INFO - [0m12:52:33  5 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_nba_news ...... [RUN]
[2024-06-20T12:52:33.625+0000] {subprocess.py:93} INFO - [0m12:52:33  5 of 8 ERROR creating sql table model NBA_STATS_dbt_nba_stats.transformed_nba_news  [[31mERROR[0m in 0.43s]
[2024-06-20T12:52:33.628+0000] {subprocess.py:93} INFO - [0m12:52:33  6 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_nba_scores .... [RUN]
[2024-06-20T12:52:34.324+0000] {subprocess.py:93} INFO - [0m12:52:34  6 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_nba_scores  [[32mSUCCESS 1[0m in 0.69s]
[2024-06-20T12:52:34.326+0000] {subprocess.py:93} INFO - [0m12:52:34  7 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_player_stats .. [RUN]
[2024-06-20T12:52:38.400+0000] {subprocess.py:93} INFO - [0m12:52:38  7 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_player_stats  [[32mSUCCESS 1[0m in 4.07s]
[2024-06-20T12:52:38.402+0000] {subprocess.py:93} INFO - [0m12:52:38  8 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_top_100_players_stats  [RUN]
[2024-06-20T12:52:42.038+0000] {subprocess.py:93} INFO - [0m12:52:42  8 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_top_100_players_stats  [[32mSUCCESS 1[0m in 3.63s]
[2024-06-20T12:52:42.042+0000] {subprocess.py:93} INFO - [0m12:52:42
[2024-06-20T12:52:42.042+0000] {subprocess.py:93} INFO - [0m12:52:42  Finished running 4 view models, 4 table models in 0 hours 0 minutes and 13.46 seconds (13.46s).
[2024-06-20T12:52:42.227+0000] {subprocess.py:93} INFO - [0m12:52:42
[2024-06-20T12:52:42.228+0000] {subprocess.py:93} INFO - [0m12:52:42  [31mCompleted with 1 error and 0 warnings:[0m
[2024-06-20T12:52:42.229+0000] {subprocess.py:93} INFO - [0m12:52:42
[2024-06-20T12:52:42.229+0000] {subprocess.py:93} INFO - [0m12:52:42    Database Error in model transformed_nba_news (models/transformed/transformed_nba_news.sql)
[2024-06-20T12:52:42.229+0000] {subprocess.py:93} INFO -   000904 (42000): SQL compilation error: error line 9 at position 8
[2024-06-20T12:52:42.230+0000] {subprocess.py:93} INFO -   invalid identifier 'NEWS_SOURCE'
[2024-06-20T12:52:42.230+0000] {subprocess.py:93} INFO -   compiled Code at target/run/nba_project/models/transformed/transformed_nba_news.sql
[2024-06-20T12:52:42.230+0000] {subprocess.py:93} INFO - [0m12:52:42
[2024-06-20T12:52:42.231+0000] {subprocess.py:93} INFO - [0m12:52:42  Done. PASS=7 WARN=0 ERROR=1 SKIP=0 TOTAL=8
[2024-06-20T12:52:42.970+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-06-20T12:52:42.971+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-20T12:52:42.998+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-06-20T12:52:43.004+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=nba_data_pipeline_dag, task_id=dbt_run, run_id=scheduled__2024-06-18T00:00:00+00:00, execution_date=20240618T000000, start_date=20240620T125224, end_date=20240620T125243
[2024-06-20T12:52:43.018+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 1063 for task dbt_run (Bash command failed. The command returned a non-zero exit code 1.; 46318)
[2024-06-20T12:52:43.045+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-20T12:52:43.077+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-20T12:52:43.084+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-20T12:57:52.694+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-20T12:57:52.740+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T12:57:52.751+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T12:57:52.751+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-06-20T12:57:52.769+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_run> on 2024-06-18 00:00:00+00:00
[2024-06-20T12:57:52.785+0000] {logging_mixin.py:188} WARNING - /home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=52295) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-20T12:57:52.786+0000] {standard_task_runner.py:63} INFO - Started process 52322 to run task
[2024-06-20T12:57:52.784+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nba_data_pipeline_dag', 'dbt_run', 'scheduled__2024-06-18T00:00:00+00:00', '--job-id', '1099', '--raw', '--subdir', 'DAGS_FOLDER/nba_data_pipeline_dag.py', '--cfg-path', '/tmp/tmp2g_87h6k']
[2024-06-20T12:57:52.789+0000] {standard_task_runner.py:91} INFO - Job 1099: Subtask dbt_run
[2024-06-20T12:57:52.899+0000] {task_command.py:426} INFO - Running <TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [running]> on host ip-172-31-45-144.ec2.internal
[2024-06-20T12:57:53.063+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='nba_data_pipeline_dag' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-18T00:00:00+00:00'
[2024-06-20T12:57:53.065+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-20T12:57:53.082+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-20T12:57:53.083+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /home/ubuntu/nba_project && dbt run']
[2024-06-20T12:57:53.099+0000] {subprocess.py:86} INFO - Output:
[2024-06-20T12:57:55.521+0000] {subprocess.py:93} INFO - [0m12:57:55  Running with dbt=1.8.2
[2024-06-20T12:57:56.602+0000] {subprocess.py:93} INFO - [0m12:57:56  Registered adapter: snowflake=1.8.3
[2024-06-20T12:57:56.924+0000] {subprocess.py:93} INFO - [0m12:57:56  Found 8 models, 4 sources, 444 macros
[2024-06-20T12:57:56.928+0000] {subprocess.py:93} INFO - [0m12:57:56
[2024-06-20T12:57:57.820+0000] {subprocess.py:93} INFO - [0m12:57:57  Concurrency: 1 threads (target='dev')
[2024-06-20T12:57:57.821+0000] {subprocess.py:93} INFO - [0m12:57:57
[2024-06-20T12:57:57.826+0000] {subprocess.py:93} INFO - [0m12:57:57  1 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_nba_news ........... [RUN]
[2024-06-20T12:57:58.383+0000] {subprocess.py:93} INFO - [0m12:57:58  1 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_nba_news ...... [[32mSUCCESS 1[0m in 0.55s]
[2024-06-20T12:57:58.385+0000] {subprocess.py:93} INFO - [0m12:57:58  2 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_nba_scores ......... [RUN]
[2024-06-20T12:58:00.005+0000] {subprocess.py:93} INFO - [0m12:58:00  2 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_nba_scores .... [[32mSUCCESS 1[0m in 1.62s]
[2024-06-20T12:58:00.008+0000] {subprocess.py:93} INFO - [0m12:58:00  3 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_player_stats ....... [RUN]
[2024-06-20T12:58:00.466+0000] {subprocess.py:93} INFO - [0m12:58:00  3 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_player_stats .. [[32mSUCCESS 1[0m in 0.46s]
[2024-06-20T12:58:00.468+0000] {subprocess.py:93} INFO - [0m12:58:00  4 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_top_100_players_stats  [RUN]
[2024-06-20T12:58:00.946+0000] {subprocess.py:93} INFO - [0m12:58:00  4 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_top_100_players_stats  [[32mSUCCESS 1[0m in 0.47s]
[2024-06-20T12:58:00.950+0000] {subprocess.py:93} INFO - [0m12:58:00  5 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_nba_news ...... [RUN]
[2024-06-20T12:58:01.422+0000] {subprocess.py:93} INFO - [0m12:58:01  5 of 8 ERROR creating sql table model NBA_STATS_dbt_nba_stats.transformed_nba_news  [[31mERROR[0m in 0.47s]
[2024-06-20T12:58:01.426+0000] {subprocess.py:93} INFO - [0m12:58:01  6 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_nba_scores .... [RUN]
[2024-06-20T12:58:02.192+0000] {subprocess.py:93} INFO - [0m12:58:02  6 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_nba_scores  [[32mSUCCESS 1[0m in 0.76s]
[2024-06-20T12:58:02.196+0000] {subprocess.py:93} INFO - [0m12:58:02  7 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_player_stats .. [RUN]
[2024-06-20T12:58:03.082+0000] {subprocess.py:93} INFO - [0m12:58:03  7 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_player_stats  [[32mSUCCESS 1[0m in 0.89s]
[2024-06-20T12:58:03.085+0000] {subprocess.py:93} INFO - [0m12:58:03  8 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_top_100_players_stats  [RUN]
[2024-06-20T12:58:04.034+0000] {subprocess.py:93} INFO - [0m12:58:04  8 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_top_100_players_stats  [[32mSUCCESS 1[0m in 0.95s]
[2024-06-20T12:58:04.038+0000] {subprocess.py:93} INFO - [0m12:58:04
[2024-06-20T12:58:04.039+0000] {subprocess.py:93} INFO - [0m12:58:04  Finished running 4 view models, 4 table models in 0 hours 0 minutes and 7.11 seconds (7.11s).
[2024-06-20T12:58:04.219+0000] {subprocess.py:93} INFO - [0m12:58:04
[2024-06-20T12:58:04.220+0000] {subprocess.py:93} INFO - [0m12:58:04  [31mCompleted with 1 error and 0 warnings:[0m
[2024-06-20T12:58:04.220+0000] {subprocess.py:93} INFO - [0m12:58:04
[2024-06-20T12:58:04.221+0000] {subprocess.py:93} INFO - [0m12:58:04    Database Error in model transformed_nba_news (models/transformed/transformed_nba_news.sql)
[2024-06-20T12:58:04.221+0000] {subprocess.py:93} INFO -   000904 (42000): SQL compilation error: error line 9 at position 8
[2024-06-20T12:58:04.221+0000] {subprocess.py:93} INFO -   invalid identifier 'NEWS_SOURCE'
[2024-06-20T12:58:04.221+0000] {subprocess.py:93} INFO -   compiled Code at target/run/nba_project/models/transformed/transformed_nba_news.sql
[2024-06-20T12:58:04.222+0000] {subprocess.py:93} INFO - [0m12:58:04
[2024-06-20T12:58:04.222+0000] {subprocess.py:93} INFO - [0m12:58:04  Done. PASS=7 WARN=0 ERROR=1 SKIP=0 TOTAL=8
[2024-06-20T12:58:05.233+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-06-20T12:58:05.234+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-20T12:58:05.261+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-06-20T12:58:05.266+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=nba_data_pipeline_dag, task_id=dbt_run, run_id=scheduled__2024-06-18T00:00:00+00:00, execution_date=20240618T000000, start_date=20240620T125752, end_date=20240620T125805
[2024-06-20T12:58:05.280+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 1099 for task dbt_run (Bash command failed. The command returned a non-zero exit code 1.; 52322)
[2024-06-20T12:58:05.291+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-20T12:58:05.323+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-20T12:58:05.325+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-20T17:44:11.169+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-20T17:44:11.226+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T17:44:11.242+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T17:44:11.242+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-06-20T17:44:11.258+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_run> on 2024-06-18 00:00:00+00:00
[2024-06-20T17:44:11.274+0000] {logging_mixin.py:188} WARNING - /home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=31926) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-20T17:44:11.276+0000] {standard_task_runner.py:63} INFO - Started process 31935 to run task
[2024-06-20T17:44:11.278+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nba_data_pipeline_dag', 'dbt_run', 'scheduled__2024-06-18T00:00:00+00:00', '--job-id', '1296', '--raw', '--subdir', 'DAGS_FOLDER/nba_data_pipeline_dag.py', '--cfg-path', '/tmp/tmpjcc4ppmd']
[2024-06-20T17:44:11.280+0000] {standard_task_runner.py:91} INFO - Job 1296: Subtask dbt_run
[2024-06-20T17:44:11.384+0000] {task_command.py:426} INFO - Running <TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [running]> on host ip-172-31-45-144.ec2.internal
[2024-06-20T17:44:11.533+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='nba_data_pipeline_dag' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-18T00:00:00+00:00'
[2024-06-20T17:44:11.534+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-20T17:44:11.546+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-20T17:44:11.547+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /home/ubuntu/nba_project && dbt run']
[2024-06-20T17:44:11.556+0000] {subprocess.py:86} INFO - Output:
[2024-06-20T17:44:13.562+0000] {subprocess.py:93} INFO - [0m17:44:13  Running with dbt=1.8.2
[2024-06-20T17:44:14.539+0000] {subprocess.py:93} INFO - [0m17:44:14  Registered adapter: snowflake=1.8.3
[2024-06-20T17:44:14.837+0000] {subprocess.py:93} INFO - [0m17:44:14  Found 8 models, 4 sources, 444 macros
[2024-06-20T17:44:14.840+0000] {subprocess.py:93} INFO - [0m17:44:14
[2024-06-20T17:44:15.801+0000] {subprocess.py:93} INFO - [0m17:44:15  Concurrency: 1 threads (target='dev')
[2024-06-20T17:44:15.801+0000] {subprocess.py:93} INFO - [0m17:44:15
[2024-06-20T17:44:15.805+0000] {subprocess.py:93} INFO - [0m17:44:15  1 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_nba_news ........... [RUN]
[2024-06-20T17:44:16.412+0000] {subprocess.py:93} INFO - [0m17:44:16  1 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_nba_news ...... [[32mSUCCESS 1[0m in 0.60s]
[2024-06-20T17:44:16.416+0000] {subprocess.py:93} INFO - [0m17:44:16  2 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_nba_scores ......... [RUN]
[2024-06-20T17:44:16.958+0000] {subprocess.py:93} INFO - [0m17:44:16  2 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_nba_scores .... [[32mSUCCESS 1[0m in 0.54s]
[2024-06-20T17:44:16.960+0000] {subprocess.py:93} INFO - [0m17:44:16  3 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_player_stats ....... [RUN]
[2024-06-20T17:44:19.731+0000] {subprocess.py:93} INFO - [0m17:44:19  3 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_player_stats .. [[32mSUCCESS 1[0m in 2.77s]
[2024-06-20T17:44:19.733+0000] {subprocess.py:93} INFO - [0m17:44:19  4 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_top_100_players_stats  [RUN]
[2024-06-20T17:44:20.318+0000] {subprocess.py:93} INFO - [0m17:44:20  4 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_top_100_players_stats  [[32mSUCCESS 1[0m in 0.58s]
[2024-06-20T17:44:20.320+0000] {subprocess.py:93} INFO - [0m17:44:20  5 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_nba_news ...... [RUN]
[2024-06-20T17:44:22.429+0000] {subprocess.py:93} INFO - [0m17:44:22  5 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_nba_news . [[32mSUCCESS 1[0m in 2.11s]
[2024-06-20T17:44:22.431+0000] {subprocess.py:93} INFO - [0m17:44:22  6 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_nba_scores .... [RUN]
[2024-06-20T17:44:23.303+0000] {subprocess.py:93} INFO - [0m17:44:23  6 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_nba_scores  [[32mSUCCESS 1[0m in 0.87s]
[2024-06-20T17:44:23.305+0000] {subprocess.py:93} INFO - [0m17:44:23  7 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_player_stats .. [RUN]
[2024-06-20T17:44:24.837+0000] {subprocess.py:93} INFO - [0m17:44:24  7 of 8 ERROR creating sql table model NBA_STATS_dbt_nba_stats.transformed_player_stats  [[31mERROR[0m in 1.53s]
[2024-06-20T17:44:24.841+0000] {subprocess.py:93} INFO - [0m17:44:24  8 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_top_100_players_stats  [RUN]
[2024-06-20T17:44:25.945+0000] {subprocess.py:93} INFO - [0m17:44:25  8 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_top_100_players_stats  [[32mSUCCESS 1[0m in 1.10s]
[2024-06-20T17:44:25.948+0000] {subprocess.py:93} INFO - [0m17:44:25
[2024-06-20T17:44:25.949+0000] {subprocess.py:93} INFO - [0m17:44:25  Finished running 4 view models, 4 table models in 0 hours 0 minutes and 11.11 seconds (11.11s).
[2024-06-20T17:44:26.082+0000] {subprocess.py:93} INFO - [0m17:44:26
[2024-06-20T17:44:26.082+0000] {subprocess.py:93} INFO - [0m17:44:26  [31mCompleted with 1 error and 0 warnings:[0m
[2024-06-20T17:44:26.083+0000] {subprocess.py:93} INFO - [0m17:44:26
[2024-06-20T17:44:26.083+0000] {subprocess.py:93} INFO - [0m17:44:26    Database Error in model transformed_player_stats (models/transformed/transformed_player_stats.sql)
[2024-06-20T17:44:26.083+0000] {subprocess.py:93} INFO -   002140 (42601): SQL compilation error:
[2024-06-20T17:44:26.084+0000] {subprocess.py:93} INFO -   Unknown function STRING_AGG
[2024-06-20T17:44:26.084+0000] {subprocess.py:93} INFO -   compiled Code at target/run/nba_project/models/transformed/transformed_player_stats.sql
[2024-06-20T17:44:26.084+0000] {subprocess.py:93} INFO - [0m17:44:26
[2024-06-20T17:44:26.084+0000] {subprocess.py:93} INFO - [0m17:44:26  Done. PASS=7 WARN=0 ERROR=1 SKIP=0 TOTAL=8
[2024-06-20T17:44:26.489+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-06-20T17:44:26.490+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-20T17:44:26.515+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-06-20T17:44:26.520+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=nba_data_pipeline_dag, task_id=dbt_run, run_id=scheduled__2024-06-18T00:00:00+00:00, execution_date=20240618T000000, start_date=20240620T174411, end_date=20240620T174426
[2024-06-20T17:44:26.532+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 1296 for task dbt_run (Bash command failed. The command returned a non-zero exit code 1.; 31935)
[2024-06-20T17:44:26.544+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-20T17:44:26.572+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-20T17:44:26.574+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-20T18:21:26.684+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-20T18:21:26.725+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T18:21:26.733+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T18:21:26.734+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-06-20T18:21:26.745+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_run> on 2024-06-18 00:00:00+00:00
[2024-06-20T18:21:26.754+0000] {logging_mixin.py:188} WARNING - /home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=54065) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-20T18:21:26.755+0000] {standard_task_runner.py:63} INFO - Started process 54073 to run task
[2024-06-20T18:21:26.754+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nba_data_pipeline_dag', 'dbt_run', 'scheduled__2024-06-18T00:00:00+00:00', '--job-id', '1435', '--raw', '--subdir', 'DAGS_FOLDER/nba_data_pipeline_dag.py', '--cfg-path', '/tmp/tmpbtv8i4ik']
[2024-06-20T18:21:26.756+0000] {standard_task_runner.py:91} INFO - Job 1435: Subtask dbt_run
[2024-06-20T18:21:26.846+0000] {task_command.py:426} INFO - Running <TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [running]> on host ip-172-31-45-144.ec2.internal
[2024-06-20T18:21:26.993+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='nba_data_pipeline_dag' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-18T00:00:00+00:00'
[2024-06-20T18:21:26.994+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-20T18:21:27.009+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-20T18:21:27.010+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /home/ubuntu/nba_project && dbt run']
[2024-06-20T18:21:27.022+0000] {subprocess.py:86} INFO - Output:
[2024-06-20T18:21:28.949+0000] {subprocess.py:93} INFO - [0m18:21:28  Running with dbt=1.8.2
[2024-06-20T18:21:29.908+0000] {subprocess.py:93} INFO - [0m18:21:29  Registered adapter: snowflake=1.8.3
[2024-06-20T18:21:30.210+0000] {subprocess.py:93} INFO - [0m18:21:30  Found 8 models, 4 sources, 444 macros
[2024-06-20T18:21:30.213+0000] {subprocess.py:93} INFO - [0m18:21:30
[2024-06-20T18:21:31.605+0000] {subprocess.py:93} INFO - [0m18:21:31  Concurrency: 1 threads (target='dev')
[2024-06-20T18:21:31.605+0000] {subprocess.py:93} INFO - [0m18:21:31
[2024-06-20T18:21:31.609+0000] {subprocess.py:93} INFO - [0m18:21:31  1 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_nba_news ........... [RUN]
[2024-06-20T18:21:32.227+0000] {subprocess.py:93} INFO - [0m18:21:32  1 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_nba_news ...... [[32mSUCCESS 1[0m in 0.61s]
[2024-06-20T18:21:32.229+0000] {subprocess.py:93} INFO - [0m18:21:32  2 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_nba_scores ......... [RUN]
[2024-06-20T18:21:37.829+0000] {subprocess.py:93} INFO - [0m18:21:37  2 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_nba_scores .... [[32mSUCCESS 1[0m in 5.60s]
[2024-06-20T18:21:37.832+0000] {subprocess.py:93} INFO - [0m18:21:37  3 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_player_stats ....... [RUN]
[2024-06-20T18:21:38.319+0000] {subprocess.py:93} INFO - [0m18:21:38  3 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_player_stats .. [[32mSUCCESS 1[0m in 0.49s]
[2024-06-20T18:21:38.322+0000] {subprocess.py:93} INFO - [0m18:21:38  4 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_top_100_players_stats  [RUN]
[2024-06-20T18:21:38.894+0000] {subprocess.py:93} INFO - [0m18:21:38  4 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_top_100_players_stats  [[32mSUCCESS 1[0m in 0.57s]
[2024-06-20T18:21:38.896+0000] {subprocess.py:93} INFO - [0m18:21:38  5 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_nba_news ...... [RUN]
[2024-06-20T18:21:39.989+0000] {subprocess.py:93} INFO - [0m18:21:39  5 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_nba_news . [[32mSUCCESS 1[0m in 1.09s]
[2024-06-20T18:21:39.992+0000] {subprocess.py:93} INFO - [0m18:21:39  6 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_nba_scores .... [RUN]
[2024-06-20T18:21:40.865+0000] {subprocess.py:93} INFO - [0m18:21:40  6 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_nba_scores  [[32mSUCCESS 1[0m in 0.87s]
[2024-06-20T18:21:40.869+0000] {subprocess.py:93} INFO - [0m18:21:40  7 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_player_stats .. [RUN]
[2024-06-20T18:21:41.297+0000] {subprocess.py:93} INFO - [0m18:21:41  7 of 8 ERROR creating sql table model NBA_STATS_dbt_nba_stats.transformed_player_stats  [[31mERROR[0m in 0.43s]
[2024-06-20T18:21:41.299+0000] {subprocess.py:93} INFO - [0m18:21:41  8 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_top_100_players_stats  [RUN]
[2024-06-20T18:21:42.398+0000] {subprocess.py:93} INFO - [0m18:21:42  8 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_top_100_players_stats  [[32mSUCCESS 1[0m in 1.10s]
[2024-06-20T18:21:42.401+0000] {subprocess.py:93} INFO - [0m18:21:42
[2024-06-20T18:21:42.402+0000] {subprocess.py:93} INFO - [0m18:21:42  Finished running 4 view models, 4 table models in 0 hours 0 minutes and 12.19 seconds (12.19s).
[2024-06-20T18:21:42.530+0000] {subprocess.py:93} INFO - [0m18:21:42
[2024-06-20T18:21:42.531+0000] {subprocess.py:93} INFO - [0m18:21:42  [31mCompleted with 1 error and 0 warnings:[0m
[2024-06-20T18:21:42.531+0000] {subprocess.py:93} INFO - [0m18:21:42
[2024-06-20T18:21:42.532+0000] {subprocess.py:93} INFO - [0m18:21:42    Database Error in model transformed_player_stats (models/transformed/transformed_player_stats.sql)
[2024-06-20T18:21:42.532+0000] {subprocess.py:93} INFO -   002061 (42601): SQL compilation error:
[2024-06-20T18:21:42.532+0000] {subprocess.py:93} INFO -   Window function type [FIRST_VALUE] requires ORDER BY in window specification.
[2024-06-20T18:21:42.532+0000] {subprocess.py:93} INFO -   compiled Code at target/run/nba_project/models/transformed/transformed_player_stats.sql
[2024-06-20T18:21:42.532+0000] {subprocess.py:93} INFO - [0m18:21:42
[2024-06-20T18:21:42.533+0000] {subprocess.py:93} INFO - [0m18:21:42  Done. PASS=7 WARN=0 ERROR=1 SKIP=0 TOTAL=8
[2024-06-20T18:21:42.955+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-06-20T18:21:42.956+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-20T18:21:42.981+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-06-20T18:21:42.985+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=nba_data_pipeline_dag, task_id=dbt_run, run_id=scheduled__2024-06-18T00:00:00+00:00, execution_date=20240618T000000, start_date=20240620T182126, end_date=20240620T182142
[2024-06-20T18:21:42.997+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 1435 for task dbt_run (Bash command failed. The command returned a non-zero exit code 1.; 54073)
[2024-06-20T18:21:43.021+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-20T18:21:43.044+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-20T18:33:43.946+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-20T18:33:44.028+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T18:33:44.049+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T18:33:44.049+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-06-20T18:33:44.063+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_run> on 2024-06-18 00:00:00+00:00
[2024-06-20T18:33:44.079+0000] {logging_mixin.py:188} WARNING - /home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=68394) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-20T18:33:44.081+0000] {standard_task_runner.py:63} INFO - Started process 68414 to run task
[2024-06-20T18:33:44.078+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nba_data_pipeline_dag', 'dbt_run', 'scheduled__2024-06-18T00:00:00+00:00', '--job-id', '1536', '--raw', '--subdir', 'DAGS_FOLDER/nba_data_pipeline_dag.py', '--cfg-path', '/tmp/tmpyc0iz7ci']
[2024-06-20T18:33:44.084+0000] {standard_task_runner.py:91} INFO - Job 1536: Subtask dbt_run
[2024-06-20T18:33:44.291+0000] {task_command.py:426} INFO - Running <TaskInstance: nba_data_pipeline_dag.dbt_run scheduled__2024-06-18T00:00:00+00:00 [running]> on host ip-172-31-45-144.ec2.internal
[2024-06-20T18:33:44.526+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='nba_data_pipeline_dag' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-18T00:00:00+00:00'
[2024-06-20T18:33:44.530+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-20T18:33:44.554+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-20T18:33:44.555+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /home/ubuntu/nba_project && dbt run']
[2024-06-20T18:33:44.572+0000] {subprocess.py:86} INFO - Output:
[2024-06-20T18:33:46.803+0000] {subprocess.py:93} INFO - [0m18:33:46  Running with dbt=1.8.2
[2024-06-20T18:33:47.791+0000] {subprocess.py:93} INFO - [0m18:33:47  Registered adapter: snowflake=1.8.3
[2024-06-20T18:33:48.160+0000] {subprocess.py:93} INFO - [0m18:33:48  Found 8 models, 4 sources, 444 macros
[2024-06-20T18:33:48.166+0000] {subprocess.py:93} INFO - [0m18:33:48
[2024-06-20T18:33:49.093+0000] {subprocess.py:93} INFO - [0m18:33:49  Concurrency: 1 threads (target='dev')
[2024-06-20T18:33:49.096+0000] {subprocess.py:93} INFO - [0m18:33:49
[2024-06-20T18:33:49.099+0000] {subprocess.py:93} INFO - [0m18:33:49  1 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_nba_news ........... [RUN]
[2024-06-20T18:33:49.758+0000] {subprocess.py:93} INFO - [0m18:33:49  1 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_nba_news ...... [[32mSUCCESS 1[0m in 0.66s]
[2024-06-20T18:33:49.760+0000] {subprocess.py:93} INFO - [0m18:33:49  2 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_nba_scores ......... [RUN]
[2024-06-20T18:33:50.222+0000] {subprocess.py:93} INFO - [0m18:33:50  2 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_nba_scores .... [[32mSUCCESS 1[0m in 0.46s]
[2024-06-20T18:33:50.224+0000] {subprocess.py:93} INFO - [0m18:33:50  3 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_player_stats ....... [RUN]
[2024-06-20T18:33:50.645+0000] {subprocess.py:93} INFO - [0m18:33:50  3 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_player_stats .. [[32mSUCCESS 1[0m in 0.42s]
[2024-06-20T18:33:50.648+0000] {subprocess.py:93} INFO - [0m18:33:50  4 of 8 START sql view model NBA_STATS_dbt_nba_stats.staging_top_100_players_stats  [RUN]
[2024-06-20T18:33:51.100+0000] {subprocess.py:93} INFO - [0m18:33:51  4 of 8 OK created sql view model NBA_STATS_dbt_nba_stats.staging_top_100_players_stats  [[32mSUCCESS 1[0m in 0.45s]
[2024-06-20T18:33:51.102+0000] {subprocess.py:93} INFO - [0m18:33:51  5 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_nba_news ...... [RUN]
[2024-06-20T18:33:52.048+0000] {subprocess.py:93} INFO - [0m18:33:52  5 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_nba_news . [[32mSUCCESS 1[0m in 0.94s]
[2024-06-20T18:33:52.051+0000] {subprocess.py:93} INFO - [0m18:33:52  6 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_nba_scores .... [RUN]
[2024-06-20T18:33:56.959+0000] {subprocess.py:93} INFO - [0m18:33:56  6 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_nba_scores  [[32mSUCCESS 1[0m in 4.91s]
[2024-06-20T18:33:56.961+0000] {subprocess.py:93} INFO - [0m18:33:56  7 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_player_stats .. [RUN]
[2024-06-20T18:33:57.710+0000] {subprocess.py:93} INFO - [0m18:33:57  7 of 8 ERROR creating sql table model NBA_STATS_dbt_nba_stats.transformed_player_stats  [[31mERROR[0m in 0.75s]
[2024-06-20T18:33:57.713+0000] {subprocess.py:93} INFO - [0m18:33:57  8 of 8 START sql table model NBA_STATS_dbt_nba_stats.transformed_top_100_players_stats  [RUN]
[2024-06-20T18:33:58.851+0000] {subprocess.py:93} INFO - [0m18:33:58  8 of 8 OK created sql table model NBA_STATS_dbt_nba_stats.transformed_top_100_players_stats  [[32mSUCCESS 1[0m in 1.14s]
[2024-06-20T18:33:58.855+0000] {subprocess.py:93} INFO - [0m18:33:58
[2024-06-20T18:33:58.855+0000] {subprocess.py:93} INFO - [0m18:33:58  Finished running 4 view models, 4 table models in 0 hours 0 minutes and 10.69 seconds (10.69s).
[2024-06-20T18:33:58.987+0000] {subprocess.py:93} INFO - [0m18:33:58
[2024-06-20T18:33:58.988+0000] {subprocess.py:93} INFO - [0m18:33:58  [31mCompleted with 1 error and 0 warnings:[0m
[2024-06-20T18:33:58.989+0000] {subprocess.py:93} INFO - [0m18:33:58
[2024-06-20T18:33:58.989+0000] {subprocess.py:93} INFO - [0m18:33:58    Database Error in model transformed_player_stats (models/transformed/transformed_player_stats.sql)
[2024-06-20T18:33:58.989+0000] {subprocess.py:93} INFO -   000979 (42601): SQL compilation error:
[2024-06-20T18:33:58.989+0000] {subprocess.py:93} INFO -   [STAGING_PLAYER_STATS.PLAYER] is not a valid group by expression
[2024-06-20T18:33:58.990+0000] {subprocess.py:93} INFO -   compiled Code at target/run/nba_project/models/transformed/transformed_player_stats.sql
[2024-06-20T18:33:58.990+0000] {subprocess.py:93} INFO - [0m18:33:58
[2024-06-20T18:33:58.991+0000] {subprocess.py:93} INFO - [0m18:33:58  Done. PASS=7 WARN=0 ERROR=1 SKIP=0 TOTAL=8
[2024-06-20T18:33:59.493+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-06-20T18:33:59.494+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-20T18:33:59.519+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/nba_ns_venv/lib/python3.12/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-06-20T18:33:59.523+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=nba_data_pipeline_dag, task_id=dbt_run, run_id=scheduled__2024-06-18T00:00:00+00:00, execution_date=20240618T000000, start_date=20240620T183344, end_date=20240620T183359
[2024-06-20T18:33:59.535+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 1536 for task dbt_run (Bash command failed. The command returned a non-zero exit code 1.; 68414)
[2024-06-20T18:33:59.555+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-20T18:33:59.583+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-20T18:33:59.585+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
